### PR TITLE
Bump *

### PR DESCRIPTION
**Pull request by benjaminapetersen**
Fix 1699372
[bugzilla 1699372](https://bugzilla.redhat.com/show_bug.cgi?id=1699372)
- openshift/api bump to pick up changes to operator config api
- kubernetes 1.13 

/assign @jhadvig @spadgett

### CODE REVIEW

**Comment by spadgett** [comment]
kube-scheduler failing. looks unrelated to the operator change

**Comment by spadgett** [comment]
```
console flag redefined: log_dir
panic: console flag redefined: log_dir

goroutine 1 [running]:
flag.(*FlagSet).Var(0xc4200de000, 0x1c874e0, 0xc4202bffa0, 0x1a8e457, 0x7, 0x1acd84c, 0x2f)
	/usr/local/go/src/flag/flag.go:810 +0x540
flag.(*FlagSet).StringVar(0xc4200de000, 0xc4202bffa0, 0x1a8e457, 0x7, 0x0, 0x0, 0x1acd84c, 0x2f)
	/usr/local/go/src/flag/flag.go:713 +0x8b
flag.(*FlagSet).String(0xc4200de000, 0x1a8e457, 0x7, 0x0, 0x0, 0x1acd84c, 0x2f, 0xc4202bff90)
	/usr/local/go/src/flag/flag.go:726 +0x8b
flag.String(0x1a8e457, 0x7, 0x0, 0x0, 0x1acd84c, 0x2f, 0xc4202e2120)
	/usr/local/go/src/flag/flag.go:733 +0x69
```

**Comment by benjaminapetersen** [comment]
Looking at that now.

**Comment by jhadvig** [comment]
logs from console operator pod
```
console flag redefined: log_dir
panic: console flag redefined: log_dir

goroutine 1 [running]:
flag.(*FlagSet).Var(0xc4200a8060, 0x1c874e0, 0xc4202b3fb0, 0x1a8e457, 0x7, 0x1acd84c, 0x2f)
	/usr/local/go/src/flag/flag.go:810 +0x540
flag.(*FlagSet).StringVar(0xc4200a8060, 0xc4202b3fb0, 0x1a8e457, 0x7, 0x0, 0x0, 0x1acd84c, 0x2f)
	/usr/local/go/src/flag/flag.go:713 +0x8b
flag.(*FlagSet).String(0xc4200a8060, 0x1a8e457, 0x7, 0x0, 0x0, 0x1acd84c, 0x2f, 0xc4202b3fa0)
	/usr/local/go/src/flag/flag.go:726 +0x8b
flag.String(0x1a8e457, 0x7, 0x0, 0x0, 0x1acd84c, 0x2f, 0xc4202d4120)
	/usr/local/go/src/flag/flag.go:733 +0x69
```

**Comment by jhadvig** [comment]
something smells fishy here :-/

**Comment by benjaminapetersen** [comment]
indeed!

**Comment by benjaminapetersen** [comment]
btw `make` works locally.

File: pkg/controller/boilerplate.go [hunk 1]
```diff
@@ -4,7 +4,7 @@ import (
 	"fmt"
```
Comment (spadgett) [inline pkg/controller/boilerplate.go]: This file looks like a duplicate of  pkg/boilerplate/controller/controller.go

Comment (benjaminapetersen) [reply pkg/controller/boilerplate.go]: Nice, this file is pretty old.

**Comment by benjaminapetersen** [comment]
@spadgett sounds good.  we have https://jira.coreos.com/browse/CONSOLE-1376 for the script cleanup.

**Comment by benjaminapetersen** [comment]
/retest

flake on extended test.

**Comment by benjaminapetersen** [comment]
Yay

**Pull request merged**